tests/vppoutputencode.cpp

```diff
@@ -103,10 +107,15 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encVideoParams.intraPeriod = encParam->intraPeriod;
     encVideoParams.ipPeriod = encParam->ipPeriod;
     encVideoParams.rcParams.bitRate = encParam->bitRate;
+    encVideoParams.rcParams.targetPercentage = encParam->targetPercentage;
```

Comment:
If the user did not set this we do not need set this.This approach we hide issues for the default setting. You always set the targePercentage, this applied to other fields like windowSize.